app/views/standup_meetings/_update.html.erb

```diff
@@ -0,0 +1,10 @@
+<%#
```

Comment:
I think it's a bit odd to have a partial named `_update.html.erb`. I would try to stick to the Rails convention for views: new, edit, show, index. If the partial is used to display an instance of a StandupMeeting, it could be called `_standup_meeting.html.erb`